en/docs/develop/product-apis/gateway-apis/gateway-v1/guide.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+WSO2 API Manager is a complete solution for publishing APIs, creating and managing a developer community, and for scalably routing API traffic. It leverages proven, production-ready, integration, security and governance components from WSO2 Enterprise Service Bus, WSO2 Identity Server, and WSO2 Governance Registry. Moreover, it is powered by WSO2 Business Activity Monitor, thereby making WSO2 API Manager ready for any large-scale deployments right away.
```

Comment:
We don't use WSO2 ESB, but we use WSO2 EI. We don't use the BAM too. This paragraph needs to be revisted and re written.

Reply:
shall we rephrase it as "WSO2 API Manager is a complete API Management solution  that addresses all aspects of API management including API development, productization, consumption, security, rate limitation, monetization and analytics. "

Reply:
https://github.com/wso2/docs-apim/pull/1527/commits/46b28b730c7fd76e644b5b021cac43b057a7b265